README.rst

```diff
@@ -82,6 +82,21 @@ You can also run linting with ``tox``:
     $ tox -e lint
 
 
+Releasing a new version
+------------------------
+
+``mozilla-django-oidc`` releases are hosted in `PyPI <https://pypi.python.org/pypi/mozilla-django-oidc>`_.
+Here are the steps you need to follow in order to push a new release:
+
+* Make sure that ``HISTORY.rst`` is up-to-date focusing mostly on backwards incompatible changes
+* Bump the project version and create a commit for that
```

Comment:
It needs to mention *where*. I believe it's `mozilla_django_oidc/__init__.py` and `setup.cfg` (what is that second file for anyway?)